components/tag-list/test/tag-list.test.js

```diff
@@ -47,17 +47,14 @@ describe('d2l-tag-list', () => {
 		].forEach(testcase => {
 			it(testcase.name, async() => {
 				const list = await fixture(basicFixture);
-				await list.updateComplete;
-				await new Promise(resolve => requestAnimationFrame(resolve));
+				await waitUntil(() => list._items, 'List items did not become ready');
 
 				const startItem = list._items[testcase.start];
-				await startItem.updateComplete;
-
 				startItem.focus();
 				dispatchKeydownEvent(startItem, testcase.key);
 
-				await new Promise(resolve => requestAnimationFrame(resolve));
-				expect(getComposedActiveElement()).to.equal(list._items[testcase.result]);
+				await nextFrame();
```

Comment:
https://open-wc.org/docs/testing/helpers/#nextframe handles the requestAnimationFrame waiting for us